autoupdater: add key for GitHub actions CI

blocktrron opened the pull request:
Keep old key off GitHub infrastructure and use a new key for signing the automated releases.

Discussed at Meeting (04.12.2023)

flokli commented:
64xxx in master and uwh branch

mweinelt commented:
Entschuldige, hätte ich ja mergen können, habe es letztlich händisch gemacht. Kudos für die Idee!

blocktrron committed:
autoupdater: add key for GitHub actions CI

Keep old key off GitHub infrastructure and use a new key for signing the
automated releases.

Discussed at Meeting (04.12.2023)

Signed-off-by: David Bauer <mail@david-bauer.net>

mweinelt commented:
This matches the `GHA_FFDA_BUILD_ECDSA_KEY_TESTING` private key?

blocktrron commented:
Not yet, but as soon as this is merged.

mweinelt commented:
Make it so.

blocktrron commented:
Deployed.
